packages/module/patternfly-docs/content/design-guidelines/design-guidelines.md

```diff
@@ -9,6 +9,22 @@ id: User feedback
 source: design-guidelines
 ---
 
-Design guidelines
+Note: User feedback lives in its own package at [@patternfly/react-user-feedback](https://github.com/patternfly/react-user-feedback).
 
-COMING SOON!
+**User feedback** is a modal component that on-premises products can use to collect asynchronous feedback from users. 
+
+## Behavior
+
+When installed, user feedback may be accessed in a couple of ways: 
+
+By selecting a masthead’s help icon and choosing “Share feedback” in the dropdown menu:
+
+![Image of expanded dropdown menu with link to sharing feedback](/packages/module/patternfly-docs/content/img/dropdown-selection.png)
+
+By selecting “Share feedback” links or buttons elsewhere in the console, such as at the end of a wizard.
+
+![Image of feedback options in a wizard](/packages/module/patternfly-docs/content/img/wizard-selection.png)
+
+By default, user feedback can point users to 2 Qualtrics forms: a [feedback form](https://console.redhat.com/self-managed-feedback-form), which asks users to offer feedback on their product experience, and a [research form]( https://console.redhat.com/self-managed-research-form), which asks users if they’d like to participate in future research with Red Hat’s User Experience Design team.
```

Comment:
```suggestion

The feedback modal includes a title, description text, an image, and one or more selectable ways to provide feedback. Each of these can be individually customized.

Each option can either link to an external URL that opens as a separate tab, or display a form in the modal that can submit its data over the network.
```

Tweaked this a bit to clarify what Don mentioned.

@dlabaj are the 3 different forms you've built (Share feedback, Report a bug, and Inform the direction of <Company>) standard, fixed things in the Extension? Or are they basically just examples of forms that people can replace with whatever kind of form they want? Just wondering if the contents of each one needs to be documented and included here or not.

And anyone please correct me if I'm wrong, but as much as I'd like to be able to point readers from Red Hat to the URLs we've standardized on internally for on-premise products to use, my understanding is that the PatternFly community site should be vendor agnostic. We may need to figure out an alternative place for Red Hatters to find these two key links in that case... (@andrew-ronaldson 👀)